### PR TITLE
Ldy

### PR DESCRIPTION
**Pull request by danyang123**
name -> value       tag -> key

Use a more standard statement：key-value pairs

### CODE REVIEW

File: @ABLITZER/ABLITZER.m [hunk 1]
```diff
@@ -40,21 +40,21 @@
 
 classdef ABLITZER < handle % Make the class a real class not a value class
 
-    properties 
+    properties
         FishStack; % Stack to store all fish data
         % Devide fishStack into different groups based on different tags
         % Put idx of data in 2nd column
-        FishGroups = struct('Name',[],'Data',[],'Tag',[],'Note',[]);
+        FishGroups = struct('Value',[],'Data',[],'Key',[],'Note',[]);
```
Comment (young24) [inline @ABLITZER/ABLITZER.m]: We'd better rephrase the "data" as "idxData".

File: @ABLITZER/findFishByKeyValuePairs.m [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
-% Find desired fish by providing tag-value pairs
-function indices = findFishByTagValuePairs(obj,varargin)
+% Find desired fish by providing key-value pairs
+
+%SYNTAX:
+%       1.  findFishByKeyValuePairs(obj,key1,value1,...)
+%       2. obj. findFishByKeyValuePairs(key1,value2,...)
+
+
+function indices = findFishByKeyValuePairs(obj,varargin)
```
Comment (young24) [inline @ABLITZER/findFishByKeyValuePairs.m:9]: Maybe "findFishByKeys" better?

File: @ABLITZER/importMatsByKeywords.m [hunk 3]
```diff
@@ -1,31 +1,32 @@
-% load mat files which matches tags provided in the same directory
-function importMatsByTags(obj, tags, pathName)
+% load mat files which matches tags provided in the same directory,,
+function importMatsByKeywords(obj, keys, pathName)
```
Comment (young24) [inline @ABLITZER/importMatsByKeywords.m]: It seems better to me that "keys -> keywords" to keep the consistency.

**Review by young24 — CHANGES_REQUESTED** [review]
Awesome, just some minor issues to improve as I commented.

File: @ABLITZER/findFishByKeyValuePairs.m (hunk 2 above)
Comment (danyang123) [reply @ABLITZER/findFishByKeyValuePairs.m:9]: Here we need to input key-value pairs , "KeyValuePairs " maybe is easier to understand ？

File: @ABLITZER/ABLITZER.m [hunk 4]
```diff
@@ -73,15 +73,15 @@
         % easy to manipulate in MATLAB
         yaml2matlab(obj, endFrame, pathName, fileName);
 
-        % load mat files which matches tags provided in the same directory
-        importMatsByTags(obj, tags, pathName);
+        % load mat files which matches keys provided in the same directory
+        importMatsByTags(obj, keys, pathName);
```
Comment (young24) [inline @ABLITZER/ABLITZER.m]: in the function name, "Tags -> keys"

**Review by young24 — COMMENTED** [review]
Good job, keep going.

**Review by young24 — APPROVED** [review]
Great, it is good to merge with the master branch.

**Pull request merged**